### ISSUE TITLE
Component: Code ContentBlock

### ISSUE THREAD

**Issue by fischeversenker** [satellytes/satellytes.com#175]
### Design
https://www.figma.com/file/FWlI63N28LP7BxqNqj1bGw/sy-004-(Design!)?node-id=3708%3A14662

### Things to consider
Syntax highlighting? What languages do we need to support? -> Which package to use?

### Expected outcome
- Clicking the "copy" button copies the content.

**Comment by georgiee** [comment]
That's a problematic component. In a perfect world I would love to have a component that does everything in the client. This would make the component very portable. Right now we rely on [gatsby-remark-prismjs](https://www.gatsbyjs.com/plugins/gatsby-remark-prismjs/) though because we rely on remark in general for the entire markdown stuff. 

I remember that I already had a working version of the client-side version running but there had been issues which made me crawl back to this solution.

Technically it should be possible to pass the content of code blocks to a component that can then parse it during runtime. To keep things simple I would rather not go this way.

Long story short, this component is actually a very thin wrapper around the content generated through `gatsby-remark-prismjs`, so we don't need to think about which syntax highlighting is supported and such. Everything related to this is already configured in the gatsby.config file.

**Comment by georgiee** [comment]
Related icon feedback story https://github.com/satellytes/satellytes.com/issues/136

**Comment by georgiee** [comment]
this won't be a component. we need to deal with that through the markdown and atomic discussion: #251
